apps/testing/nested/src/app/child.component.spec.ts

```diff
@@ -1,16 +1,50 @@
-import { render } from '@testing-library/angular';
+import { userEvent } from '@testing-library/user-event';
+import { render, screen } from '@testing-library/angular';
 import { ChildComponent } from './child.component';
+import { createMockWithValues } from '@testing-library/angular/jest-utils';
+import { HttpService } from './http.service';
 
 describe('ChildComponent', () => {
+  const setup = async () => {
+    const httpServiceMock = createMockWithValues(HttpService, {
+      sendTitle: jest.fn(),
+    });
+    await render(ChildComponent, {
+      componentProviders: [
```

Comment:
it should be on the provider property

componentProviders is only to override service provider at component level

Reply:
i have changed this to providers